app/main.py

```diff
@@ -1,3 +1,58 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        else:
+            self.km += other
+            return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other):
+        if isinstance(other, Distance):
+            return True if self.km < other.km else False
+        else:
+            return True if self.km < other else False
+
+    def __gt__(self, other):
+        if isinstance(other, Distance):
+            return True if self.km > other.km else False
+        else:
+            return True if self.km > other else False
+
+    def __eq__(self, other):
+        if isinstance(other, Distance):
+            return True if self.km == other.km else False
+        else:
+            return True if self.km == other else False
+
+    def __le__(self, other):
+        return True if self.__lt__(other) or self.__eq__(other) \
+            else False
```

Comment:
Good direction but for __le__ opposite is
not __gt__ 
so rewrite with one another func